### PR TITLE
COmanage_tmp_directory_permissions

### PR DESCRIPTION
**Pull request by ioigoume**
COmanage tmp directory permissions

### CODE REVIEW

File: roles/comanage-registry/tasks/install.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@
     state: directory
     owner: "{{ comanage_webserver_user }}"
     group: "{{ comanage_webserver_group }}"
-    mode: 0701
+    mode: 0700
```
Comment (NicolasLiampotis) [inline roles/comanage-registry/tasks/install.yml:52]: set owner + group to webserver

File: roles/comanage-registry/tasks/install.yml [hunk 2]
```diff
@@ -65,6 +65,6 @@
     src: "{{ comanage_temp_dir }}"
     dest: "{{ comanage_path }}/local/tmp"
     state: link
-    owner: root
-    group: root
+    owner: "{{ comanage_webserver_user }}"
+    group: "{{ comanage_webserver_group }}"
```
Comment (NicolasLiampotis) [inline roles/comanage-registry/tasks/install.yml:69]: revert owner + group to root

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
- cache directory should be owned by webserver
- link to cache directory should be owned by root

File: roles/comanage-registry/tasks/install.yml (hunk 2 above)
Comment (ioigoume) [reply roles/comanage-registry/tasks/install.yml]: done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**